Porting/sync-with-cpan

```
@@ -162,6 +162,7 @@ use autodie;
 require "./Porting/Maintainers.pl";
 
 my $MAKE_LOG = 'make.log';
+unlink $MAKE_LOG if -e $MAKE_LOG;
```

does it need the test? just unlink it and ignore any error.

---

the code uses autodie. If it errors out it will die.

---

FWIW, what you suggested is what i did originally. I am not used to using autodie. :-)